text/src/autogluon/text/automm/configs/data/default.yaml

```diff
@@ -1,5 +1,6 @@
 data:
   image:
+    use_zero_img: False  # Whether to use a zero image if an image is missing.
```

Comment:
How about call it `missing_value_strategy`?

Reply:
Done.